bindings/matrix-sdk-ffi/src/timeline/mod.rs

```diff
@@ -99,7 +99,7 @@ impl Timeline {
         unsafe { Arc::from_raw(Arc::into_raw(inner) as _) }
     }
 
-    fn send_attachment(
+    async fn send_attachment(
```

Comment:
Looks like this doesn't need to be async anymore? (and ditto for callers)

Reply:
🤦‍♂️ Sorry, yes.